assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -42,6 +43,12 @@ function CanvasLayout() {
 			<Area area="page">
 				<Page />
 			</Area>
+			<Area area="prev">
+				<PageNav direction="prev" />
```

Comment:
Is a boolean better? Maybe two booleans, `isNext` and `isPrev` both defaulting to `false`?

Reply:
Changed to using just `isNext`, default to `true`, seemed to be more straightforward than using two booleans, LMK if you think otherwise.